### ISSUE TITLE
about data_pipe.py issues

### ISSUE THREAD

**Issue by konioyxgq** [ZhaoJ9014/face.evoLVe.PyTorch#113]
In data_pipe.py ,**load_mx_rec** function
        header, img = mx.recordio.unpack_img(img_info)### **img is BGR**
        label = int(header.label)
        img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR) ### **img is RGB**
        img = Image.fromarray(img)
so train data is RGB.
But ,in **load_bin** function
        img = mx.image.imdecode(_bin).asnumpy() ### **img is RGB**
        img = cv2.cvtColor(img, cv2.COLOR_RGB2BGR) ### **img is BGR**
so test data is BGR.

Training data and test data formats are different.

**Comment by konioyxgq** [comment]
@ZhaoJ9014 Please!  Help me! Did I get it wrong?

**Comment by konioyxgq** [comment]
HELLO！！ Can someone help me? @verasativa @clhne @insightcs @ZhaoJ9014

**Comment by konioyxgq** [comment]
Excuse me.could you give me a hand? @verasativa @clhne @insightcs @ZhaoJ9014

**Comment by verasativa** [comment]
I'm sorry, after my contribs to this repo, I got really worried that it may be used against oppressed minorities by totalitarian states, like Uyghurs by ccp. Not wanted to take the risk of helping that kind of evil, I decided not to involve in any biometrics stuff any more.

**Comment by konioyxgq** [comment]
Excuse me.
